lms/djangoapps/course_api/api.py

```diff
@@ -0,0 +1,58 @@
+"""
+Course API
+"""
+
+from django.contrib.auth.models import User
+from lms.djangoapps.courseware.courses import (
+    get_courses,
+    get_course_by_id
+)
+from opaque_keys.edx.keys import CourseKey
+
+from .serializers import CourseSerializer
+
+
+def _has_permission(requesting_user, username):
```

Comment:
to conform for standard project structure, please move this to a permissions.py file.